find_green_build

```diff
@@ -39,9 +40,13 @@ PROG=${0##*/}
 #+ OPTIONS
 #+     [--official]          - Produce an official release version for a
 #+                             release branch
-#+     --github-token=       - Must be specified if GITHUB_TOKEN not set
-#+     --exclude-suites=     - Space separated list of CI suites to exclude
+#+     [--github-token=]     - Must be specified if GITHUB_TOKEN not set
+#+     [--exclude-suites=]   - Space separated list of CI suites to exclude
 #+                             from go/nogo criteria
+#+     [--allow-stale-build] - Keep going even if the build is behind HEAD,
+#+                             just for informational purposes.
```

Comment:
Maybe also state that branch releases cannot run against anything but HEAD.
And with that change I might also change the "GIVE UP" text to FAILED as "GIVE UP" implies that we're stopping there.